### ISSUE TITLE
After Update to 8.1.1  Visited / Read forums icons not reflecting actual status anymore.

### ISSUE THREAD

**Issue by gnossosp** [DNNCommunity/Dnn.CommunityForums#973]
## Describe the bug
A "Read (rhymes with Red)" forum not rendering 'Read Forum' font/folder icon

## Software Versions
- DNN:  09.13.03
- Module: 8.1.1

## To Reproduce
Updated Extension from 8.01 to 8.01.01 ... Re Indexed "Search" as suggested.
Open forum and all folders are now showing "Unread" state folder Icon.

## Expected behavior
Folder icon for Read Forums should show the Read Icon (screen caps and class below)

## Screenshots

All topics have been "read" in this forum still showing "unread" icon

CSS rendered unread content. Doesn't change as expected
< i class="fa fa-folder fa-red fa-2x"></i >
![Clipboard_07-05-2024_01-forum](https://github.com/DNNCommunity/Dnn.CommunityForums/assets/72934916/ce65fe62-4867-47bd-b1d0-d50951d8e7e0)

Empty icon:
< i class="fa fa-folder-o fa-grey fa-2x"></i >
(not shown in screen caps for brevity)

I _manually_ changed Class to show expected icon for "read" folders Please ignore the actual content stuff:
< i class="fa fa-folder-o fa-red fa-2x"></i >
Should show the "read status" Icon for  Sample Forum 3
![Clipboard_07-05-2024_0102](https://github.com/DNNCommunity/Dnn.CommunityForums/assets/72934916/8016c42d-06ff-4141-ad64-4de498c11b4d)

The Empty forums are still rendering the grey outlined folder icon/font as expected:
< i class="fa fa-folder-o fa-grey fa-2x"></i >

## Error log
**Note: Debug DLL's**
Please replace the current extension DLL's with the debug DLL's (if these are available with the release) and reproduce the error with the debug DLL's before pasting the error log.

Paste the error log that is related to this issue.

**Comment by johnhenley** [comment]
@gnossosp thanks for upgrading :). The page is cached for a couple minutes so try a refresh and see if they change.

**Comment by gnossosp** [comment]
> @gnossosp thanks for upgrading :). The page is cached for a couple minutes so try a refresh and see if they change.

@johnhenley - Thanks for y'alls hard work on this!

It's been over 20 minutes since I posted error.

 I cleared site server cache and restarted DNN application. Opened a different browser  (FF) with completely cleared history.  Logged on, opened forum, marked "all read". I even manually read each of the three topics in the forum...
 
 Initially I installed the update on a different live instance that I don't use publicly -  you know I'm cautious - completely different server. Waited over an  hour, cleared DNN cache, restated instance. Used clean (cleared history) browser. 

Same result. Sorry.
